src/applications/disability-benefits/all-claims/config/form.js

```diff
@@ -186,6 +189,27 @@ const formConfig = {
           onContinue: captureEvents.militaryHistory,
           appStateSelector: state => ({ dob: state.user.profile.dob }),
         },
+        verifyBdd: {
+          title: 'Verify active duty status',
+          path: 'review-veteran-details/verify-bdd',
```

Comment:
@katelyncai @goldenmeanie Do you have opinions about the URLs to these three new pages?